index.html

```diff
@@ -0,0 +1,232 @@
+                    <li><a href="#"><img src="./images/Linkedin icon.svg" alt="Linkedin"></a></li>
+                    <li><a href="#"><img src="./images/angellist.svg" alt="Enabled"></a></li>
+                    <li><a href="#"><img src="./images/twitter.svg" alt="Twitter"></a></li>
+                    <li><a href="#"><img src="./images/medium.svg" alt="Medium"></a></li>
+                </ul>
+            </div>
+            <div class="scroll">
+                <img src="./images/Disabled.png" alt="scroll button">
+            </div>
+        </div>
+    </section>
+
+    <section class="works">
+       <div class="works-flex">
+        <div class="recent-works">
+            <h2>My Recent Works</h2>
+            <img class="indicator" src="./images/Indicator.svg" alt="Indicator">
+         </div>
+         <div class="individual-card-item-grid">
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
+                     <button type="button" class="wrapper"><a href="#">See project</a></button>
+                 </article>
+             </article>
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
+                     <button type="button" class="wrapper"><a href="#">See project</a></button>
+                 </article>
+             </article>
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
+                     <button type="button" class="wrapper"><a href="#">See project</a></button>
+                 </article>
+             </article>
+             <article class="card-work-item">
+                 <article class="individual-card-item">
+                     <h3>Multi-Post Stories <br/> Gain+Glory</h3>
+                     <ul class="languages">
+                         <li>Ruby on Rails</li>
+                         <li>CSS</li>
+                         <li>JavaScript</li>
+                         <li>HTML</li>
+                     </ul>
```

Comment:
Can you kindly remove this  black border-color you introduced to the button of your project

![image](https://github.com/zhorabay/My-portfolio/assets/20537712/9836b3cf-ccd0-42eb-a4f9-a8272c0c2392)